### PR TITLE
chore(deps): upgrade cucumber v8

### LINKED ISSUE #88: Upgrade cucumber to v8.x
Cucumber.js released version v8 with some breaking changes. We should upgrade cucumber and make it compatible for veggies.

### PR DESCRIPTION
**Pull request by tienduy-nguyen**
Close #88

### CODE REVIEW

**Comment by Crow-EH** [comment]
You shouldn't commit `tests/cli/dummy_features/__snapshots__/clean_snapshots.feature.snap` and `tests/cli/dummy_features/__snapshots__/update_snapshots.feature.snap`.

Otherwise, LGTM 👍

**Comment by tienduy-nguyen** [comment]
> You shouldn't commit `tests/cli/dummy_features/__snapshots__/clean_snapshots.feature.snap` and `tests/cli/dummy_features/__snapshots__/update_snapshots.feature.snap`: They are supposed to be changed by the tests, but should be left as is for the tests to be useful.
> 
> Otherwise, LGTM 👍

Thank Yohan so much ❤️. I waited for this review a long time 😂. You are the best.

(I don't know we always want to remain theses snapshots, so I commit them after run the command test-cli. Ok I see the thing now. They are already restored).

**Comment by Crow-EH** [comment]
Great.

I've been ousted from the @ekino group so you'll have to ask @pebie to merge

File: package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
   },
   "devDependencies": {
     "@commitlint/config-conventional": "12.x",
-    "@cucumber/cucumber": "7.x",
+    "@cucumber/cucumber": "8.x",
```
Comment (pebie) [inline package.json:47]: You forgot `peerDependencies` L67

**Comment by pebie** [comment]
Why is the yarn interactive tools plugin installed ?

**Comment by tienduy-nguyen** [comment]
> Why is the yarn interactive tools plugin installed ?

I think this plugin could be very useful. When I started to do this task, I want to create  another commit to upgrade other dependencies. Some test failed and it need some modifications for the compatibility.

But if I do it, it make so much noise for the task cucumber v8. I will take that later.

So maybe better I will remove this plugin from this PR.

**Comment by pebie** [comment]
> > Why is the yarn interactive tools plugin installed ?
> 
> I think this plugin could be very useful. When I started to do this task, I want to create another commit to upgrade other dependencies. Some test failed and it need some modifications for the compatibility.
> 
> But if I do it, it make so much noise for the task cucumber v8. I will take that later.
> 
> So maybe better I will remove this plugin from this PR.

Yes do another PR which explains why do you need this plugin. In this one there is no need to install this plugin. So yes, it makes noise.

**Comment by tienduy-nguyen** [comment]
> > > Why is the yarn interactive tools plugin installed ?
> > 
> > 
> > I think this plugin could be very useful. When I started to do this task, I want to create another commit to upgrade other dependencies. Some test failed and it need some modifications for the compatibility.
> > But if I do it, it make so much noise for the task cucumber v8. I will take that later.
> > So maybe better I will remove this plugin from this PR.
> 
> Yes do another PR which explains why do you need this plugin. In this one there is no need to install this plugin. So yes, it makes noise.

Ok, agree with that. I removed all the changes of yarn.yaml and plugins. I'll create another PR later.

**Pull request merged**